interceptors/logging/payload.go

```diff
@@ -136,6 +135,7 @@ func logProtoMessageAsJson(logger Logger, pbMsg interface{}, key string, msg str
 		payload, err := (&jsonpbObjectMarshaler{pb: p}).marshalJSON()
 		if err != nil {
 			logger = logger.With(key, err.Error())
+			logger.Log(ERROR, err.Error())
```

Comment:
This change seems unrelated, is it a bug fix?

Reply:
Ohh my bad, I was debugging the failing tests and added this line. Will remove this.